run/GCHP/HISTORY.rc.templates/HISTORY.rc.carbon

```diff
@@ -206,6 +206,10 @@ COLLECTIONS: 'Emissions',
                               'BudgetConvectionTrop_CH4      ', 'GCHPchem',
                               'BudgetConvectionPBL_CH4       ', 'GCHPchem',
                               'BudgetConvectionLevs1to35_CH4 ', 'GCHPchem',
+                              'BudgetChemistryFull_CH4      ', 'GCHPchem',
```

Comment:
Thanks for catching that this was missing. I assume we didn't have chemistry budget diagnostics for carbon simulation because we did not use KPP. @msulprizio or @yantosca, could you confirm that was the case?